tests/testthat/helper-test-dir.R

```diff
@@ -29,3 +29,40 @@ local_stan_bern_model <- function(clean = TRUE, .local_envir = parent.frame()) {
   copy_model_from(STAN_MOD3, file.path(mdir, "bern"))
   withr::local_dir(tdir, .local_envir = .local_envir)
 }
+
+#' Create temporary directory with a dummy nmbayes model
```

Comment:
It looks like this will be useful for the `check_nonmem_finished` tests too.  I'm going to split it off into its own commit at the front of this series so that I can pull it into the other series as well.